backend/tracim_backend/lib/search/elasticsearch_search/elasticsearch_search.py

```diff
@@ -186,6 +186,78 @@ def delete_indices(self) -> None:
+        author = self._create_digest_user_from_user(content_in_context.author)
+        last_modifier = self._create_digest_user_from_user(content_in_context.last_modifier)
+        workspace = DigestWorkspace(
+            workspace_id=content_in_context.workspace.workspace_id,
+            label=content_in_context.workspace.label,
+        )
+        path = [
+            DigestContent(
+                content_id=component.content_id,
+                label=component.label,
+                slug=component.slug,
+                content_type=component.content_type,
+            )
+            for component in content_in_context.content_path
+        ]
+        indexed_subcontent = self._index_subcontents(content_in_context)
+        comments = indexed_subcontent["comments"]
+        todos = indexed_subcontent["todos"]
+        tags = [content_tag.tag.tag_name for content_tag in content.tags]
+        indexed_content = IndexedContent(
+            content_namespace=content_in_context.content_namespace,
+            content_id=content_in_context.content_id,
+            current_revision_id=content_in_context.current_revision_id,
+            current_revision_type=content_in_context.current_revision_type,
+            slug=content_in_context.slug,
+            parent_id=content_in_context.parent_id,
+            workspace_id=content_in_context.workspace_id,
+            workspace=workspace,
+            label=content_in_context.label,
+            content_type=content_in_context.content_type,
+            sub_content_types=content_in_context.sub_content_types,
+            status=content_in_context.status,
+            is_archived=content_in_context.is_archived,
+            is_deleted=content_in_context.is_deleted,
+            is_editable=content_in_context.is_editable,
+            is_active=content_in_context.is_active,
+            show_in_ui=content_in_context.show_in_ui,
+            file_extension=content_in_context.file_extension,
+            filename=content_in_context.filename,
+            modified=content_in_context.modified,
+            created=content_in_context.created,
+            active_shares=content_in_context.actives_shares,
+            description=content_in_context.description,
+            path=path,
+            comments=comments,
+            comment_count=len(comments),
+            todos=todos,
+            todo_count=len(todos),
+            tags=tags,
+            tag_count=len(tags),
+            author=author,
+            last_modifier=last_modifier,
+            archived_through_parent_id=content_in_context.archived_through_parent_id,
+            deleted_through_parent_id=content_in_context.deleted_through_parent_id,
+            raw_content=content_in_context.raw_content,
+            content_size=content_in_context.size,
+        )
+        indexed_content.meta.id = content_in_context.content_id
+        content_index_alias = self._get_index_parameters(IndexedContent).alias
```

Comment:
what is an alias?

Reply:
the alias in that case should be "-content"